paddle/operators/functor/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+if(WITH_GPU)
```

Comment:
It looks an overkill to create a subdirectory and a namespace for just one class `Set`?